extensions.csproj

```diff
@@ -1,13 +1,9 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>netcoreapp3.1</TargetFramework>
-    <AzureFunctionsVersion>v3</AzureFunctionsVersion>
-	<WarningsAsErrors></WarningsAsErrors>
-	<DefaultItemExcludes>**</DefaultItemExcludes>
+    <TargetFramework>net6.0</TargetFramework>
+    <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <WarningsAsErrors></WarningsAsErrors>
+    <DefaultItemExcludes>**</DefaultItemExcludes>
   </PropertyGroup>
-  <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.DurableTask" Version="2.4.2" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.Storage" Version="4.0.2" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Script.ExtensionsMetadataGenerator" Version="1.1.8" />
```

Comment:
Is safe remove all the packages from `extensions.csproj`? The project works even on local execution?

Reply:
an azure function has been run without those lines and using a testing cosmos db of our infrastructure.

i removed those lines because explicitly declaring and using an extension bundle is mandatory in functions runtime v4, therefore removing the needs for those lines. the drawback i see is that if microsoft has a bug on a latest version of an extension, you can't go back until they update.